tripy/nvtripy/frontend/ops/plugin.py

```diff
@@ -75,3 +76,122 @@ def plugin(
         output_info,
         kwargs,
     )
+
+@export.public_api(document_under="operations/functions")
+def plugin(
+    op: trtp._lib.PluginDef,
```

Comment:
Would it be possible to use the same string that we register the plugin with, i.e. `"example::elemwise_add_plugin"`? We can use `getattr` internally to grab the correct plugin from TRT. This way we keep TRT types out of the Tripy interface.